### ISSUE TITLE
📚 Documentation: multiple <nav> elements should be labelled [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#294]
## Issue
There are multiple `<nav>` elements on each page.  When you have more than one `<nav>` you should add a label to each.

## Why it matters
When you have more than one `<nav>` element you should add a label to each to aid navigation with assistive tech and to differentiate them, otherwise when navigate via landmarks it can be confusing which navigation a user is targetting.

## Suggested fix
Add `aria-label`s to each of the `<nav>` elements to identify their purposes. 

For example:
Nav at the top could be `<nav aria-label="main">` and the nav at the side could be `<nav aria-label="docs">` (these could be better, if someone can think of more descriptive names for each navigation element. You do not need to say "navigation" as part of the text.)

## Relevant Guidance
[multiple navigation landmarks](https://www.aditus.io/patterns/multiple-navigation-landmarks/)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by divyanshkul** [comment]
Would love to work on this!
Also, could this be added to Appwrite's Hacktoberfest program?

**Comment by TGlide** [comment]
@divyanshkul Assigned it to you. We can add the `hacktoberfest-accepted` label to a merged PR 😄

**Comment by CBID2** [comment]
Hi @TGlide. Hacktoberfest is over, and it seems that the assignee has not worked on this issue. If you're open, I'd love to work on this.

**Comment by TGlide** [comment]
> Hi @TGlide. Hacktoberfest is over, and it seems that the assignee has not worked on this issue. If you're open, I'd love to work on this.

Go ahead!

**Comment by guptadeepak8** [comment]
@CBID2  are you working on this issue?

**Comment by CBID2** [comment]
> @CBID2  are you working on this issue?

Yes

**Comment by guptadeepak8** [comment]
Great👍

**Comment by CBID2** [comment]
Hi @TGlide! :) I made the PR for this. It's #358
